db/agendadb/db.go

```diff
@@ -220,13 +220,20 @@ func GetAllAgendas() (agendas []*AgendaTagged, err error) {
 	adb, err := Open(dbName)
 	if err != nil {
 		log.Errorf("Failed to open new DB: %v", err)
+		return
 	}
+
+	defer func() {
+		err = adb.Close()
+		if err != nil {
+			log.Errorf("Failed to close the DB: %v", err)
```

Comment:
Please mention agenda DB, otherwise it could be confusing what threw the error.

Reply:
Ok. Noted